tikv-client/src/test/java/com/pingcap/tikv/codec/TableCodecV2Test.java

```diff
@@ -387,11 +393,49 @@ private void debug(Object[] a1, Object[] a2) {
       System.out.println("Expected=[" + toString(a1) + "]\nResult  =[" + toString(a2) + "]");
     }
 
+    private void debug(byte[] a1, byte[] a2) {
+      System.out.println(
+          "Expected=[" + Arrays.toString(a1) + "]\nResult  =[" + Arrays.toString(a2) + "]");
+    }
+
     private void testDecode() {
       Row res = TableCodecV2.decodeRow(this.bytes, this.handle, this.tableInfo);
       Object[] o = fromRow(res, this.tableInfo);
       debug(this.value, o);
       assertTrue(deepEquals(this.value, o));
     }
+
+    private void testEncode() {
+      byte[] bytes =
+          TableCodecV2.encodeRow(
+              this.tableInfo.getColumns(), this.value, this.tableInfo.isPkHandle());
+      debug(this.bytes, bytes);
+      if (this.tableInfo
+          .getColumns()
+          .stream()
+          .noneMatch(
+              x -> {
+                switch (x.getType().getType()) {
+                  case TypeFloat:
+                    // double value may have precision loss
+                    return true;
+                  default:
+                    return false;
+                }
+              })) {
+        assertTrue(equals(this.bytes, bytes));
+      } else {
+        // FIXME: temporary code just for reverse checking...
+        Row res = TableCodecV2.decodeRow(bytes, this.handle, this.tableInfo);
+        Object[] o = fromRow(res, this.tableInfo);
+        debug(this.value, o);
+        assertTrue(deepEquals(this.value, o));
+      }
```

Comment:
Added this check because for float/double like `11.99`, its binary representation may not be consistent in IEEE 754 format due to precision loss.

Reply:
... so if float/double value exists in table info, we reverse check the result by first encode and then decode to see if the answer still matches.

Reply:
After fixing float encoding, these lines are not needed.